indico/web/flask/blueprints/legacy.py

```diff
@@ -55,8 +55,8 @@
     'adminLayout-setDefaultPDFOptions', 'adminLayout-styles', 'adminList',
     'adminList-switchNewsActive', 'adminMaintenance', 'adminMaintenance-pack', 'adminMaintenance-performPack',
     'adminMaintenance-performTmpCleanup', 'adminMaintenance-tmpCleanup', 'adminProtection', 'adminServices-ipbasedacl',
-    'adminServices-ipbasedacl_fagrant', 'adminServices-ipbasedacl_farevoke', 'adminServices-oauthAuthorized',
```

Comment:
I know, but since I was removing all references to legacy stuff it didn't hurt to remove a couple of lines. :P